x-pack/plugins/synthetics/public/apps/synthetics/state/monitor_management/monitor_list.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { takeLeading } from 'redux-saga/effects';
+import {
+  FetchMonitorManagementListQueryArgs,
+  MonitorManagementListResult,
+  MonitorManagementListResultCodec,
+} from '../../../../../common/runtime_types';
+import { SyntheticsAppState } from '../root_reducer';
```

Comment:
Because root_reducer also has to import this slice, it might be a cyclic import. A separate selectors.ts file would help if so.